### PR TITLE
Christanbul

### PR DESCRIPTION
**Pull request by chrisw310**
DECO1400, 1800, 2300, 3500

### CODE REVIEW

File: tex/courses/subjects/DECO1400.tex [hunk 1]
```diff
@@ -2,16 +2,31 @@
...
 contact = {2L, 1T, 2P},
 coordinator = {Ms Lorna MacDonald (lorna@itee.uq.edu.au)},
 assessment = {
-example & example\% & example \\
-},
+Tutorials & 20\% & Randomly picked presentations each week, and a paper prototype \\
+Practicals & 30\% & Two exams: one on HTML/CSS, and one on jQuery \\
+Project & 25\% & A fully implemented website in HTML/CSS, plus documentation \\
+Final Exam & 25\% & Closed book final exam on lecture material \\
+
```
Comment (wisebaldone) [inline tex/courses/subjects/DECO1400.tex]: missing `},`

File: tex/courses/subjects/DECO1400.tex [hunk 2]
```diff
@@ -2,16 +2,31 @@
...
 review = {
-    review here
+    This course will teach you all you need to know to create a basic website with HTMl, CSS and JavaScript. While this course has become more theoretical recently, it still a very practical course.
+
```
Comment (wisebaldone) [inline tex/courses/subjects/DECO1400.tex:17]: missing `\\`

Comment (wisebaldone) [reply tex/courses/subjects/DECO1400.tex:17]: :taco: for the other paragraphs aswell.

File: tex/courses/subjects/DECO1800.tex [hunk 3]
```diff
@@ -3,15 +3,29 @@
...
+Project Part B & 15\% & Minimal implementation of the project and a document outlining changes made and group planning \\
+Project Part C & 30\% & Final design and a document outlining final design decisions and reflection \\
+Design Portfolio & 20\% & A full HTML website explaining the project process, the portfolio process, and a course reflection \\
+Reflective Journal & 20\% & A weekly blog of progress made and lessons learned in the course \\
 },
 review = {
-    review here
+    Contrary to what you may think, DECO1800 is more of a course which teaches how to work in groups, rather than a course on web design skills. You will be placed into a group of 4 and you will be required to create a website which makes use of the Trove database. Unfortunately, group work projects are a bit of a 'dice roll' in who you are going to get, but this course aims to teach you how to work within groups to create a quality project. In some ways this course is like ENGG2800, but for IT students. The group work emphasis is made evident with the constant documentation and blogging of lessons you have learnt, and suggestions to improve the design process. 
+
```
Comment (wisebaldone) [inline tex/courses/subjects/DECO1800.tex:18]: see :taco:

File: tex/courses/subjects/DECO2300.tex [hunk 4]
```diff
@@ -2,16 +2,30 @@
...
+ Video Prototype & 18.75\% & A short video explaining the concept for your prototype, and a statement of purpose. \\
+ Prototype 1 & 18.75\% & A digital prototype made in Unity, and a SOP \\
+ Prototype 2 & 18.75\% & A physical prototype made with the MakeyMakey, and a SOP \\
+ Prototype 3 & 18.75\% & An iteration of the prototype which explores the prototype beyond the original concept, and a SOP \\
+ Blog & 25\% & A weekly blog consisting of responses to contact classes, and an explanation of the process of creating your prototype. \\
 },
 review = {
-    review here
+    This course is all about prototyping designs. This will involve using the MakeyMakey and Unity to create new ideas for interfaces which will need testing and iteration.
+
```
Comment (wisebaldone) [inline tex/courses/subjects/DECO2300.tex:18]: see :taco:

File: tex/courses/subjects/DECO3500.tex [hunk 5]
```diff
@@ -2,16 +2,31 @@
...
+Project Idea & 10\% & A poster explaining your concept \\
+Project Plan & 20\% & A document explaining the problem space and the concept \\
+Project Implementation & 15\% & A prototype which demonstrates key interactions of the product \\
+Project Standup & 15\% & Three fortnightly speeches of the progress made on the project worth 5\% each  \\
+Promotional Material & 10\% & A poster, video or website explaining the concept to a wider audience \\
+Reflection Essay & 30\% & An essay reflecting on the project and how it links to the lecture topics \\
 },
 review = {
-    review here
+    'Social \&amp; Mobile Computing' is a course designed to practially apply the skills learnt in DECO2500 to a real project. This involves defining a problem in journalism, and proposing a solution by building a digital prototype. It is required that students will perform usability tests and iterate on their designs from the feedback.
```
Comment (wisebaldone) [inline tex/courses/subjects/DECO3500.tex]: see :taco:

**Comment by TRManderson** [comment]
Are we still waiting on :taco: @wbo @chrisw310?

**Comment by wisebaldone** [comment]
Yep, I'm fine to let this through and patch it (simple fixes) if you think the content is sensible.

**Comment by TRManderson** [comment]
LGTM, though I haven't done any of the courses listed

**Comment by wisebaldone** [comment]
taking toms LGTM as the third :+1:

**Pull request merged**